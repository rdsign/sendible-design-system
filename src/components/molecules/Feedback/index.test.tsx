import { create } from 'react-test-renderer';
import { Feedback } from '.';
import { Template } from '../../../../spec/template';

describe('Feedback', () => {
    const component = (
        <Template>
            <Feedback icon="danger" message="This is a generic message." />
        </Template>
    );

    test('should match snapshot', () => {
        const snapshot = create(component);
        expect(snapshot).toMatchSnapshot();
    });
});
