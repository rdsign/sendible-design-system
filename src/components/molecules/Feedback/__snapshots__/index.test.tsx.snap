// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Feedback should match snapshot 1`] = `
.c1 {
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  fill: #201E24;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  height: 1.2rem;
  width: 1.2rem;
}

.c1 svg {
  height: 100%;
  width: 100%;
}

.c0 {
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  color: #CE091C;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  font-size: 1.2rem;
  font-weight: 500;
  line-height: 1em;
  -webkit-user-select: none;
  -moz-user-select: none;
  -ms-user-select: none;
  user-select: none;
}

.c0 div {
  fill: #E50A1F;
  margin-right: 0.5rem;
}

<span
  className="c0"
>
  <div
    className="c1"
    name="danger"
    size="small"
  >
    <svg
      data-testid="svg"
      height="24"
      viewBox="0 0 24 24"
      width="24"
      xmlns="http://www.w3.org/2000/svg"
      xmlnsXlink="http://www.w3.org/1999/xlink"
    />
  </div>
  This is a generic message.
</span>
`;
